packages/preview-middleware/src/base/telemetry-reporter.ts

```diff
@@ -15,14 +12,10 @@ export class TelemetryReporter {
     constructor(private eventName: string, private readonly logger: Logger, private layer?: string) {}
     public async initializeTelemetry() {
         try {
-            // const envFilePath = { path: join(__dirname, '..', '..', '.env') };
-            // dotenv.config(envFilePath);
-            const internalFeature = isInternalFeaturesSettingEnabled();
             await initTelemetrySettings({
                 consumerModule: modulePackageJson,
-                internalFeature,
+                internalFeature: false,
```

Comment:
Should this value be hard coded? We won't be able to change this even if it is used in an internal environment.

Reply:
So far i have observed is for cpe and adp it was always started from cli and the method that check relies on vscodehttps://github.com/SAP/open-ux-tools/blob/dbae4d0a5d00e0e89961df1fdccfbe3559326614/packages/feature-toggle/src/featureToggle.ts#L135 and this env variable and which is not currnely used by our module so to avoid deps with FeatureToggle package i suggested that it can be always set false

Reply:
I have added it back. thanks

Reply:
@voicis  @Jimmy-Joseph19 

Internal feature toggle was not part of open source library. Therefore telemetry lib was not able to call feature toggle internally like we did in internal repo. That's why it was exposed as a configuration and let the consumer to pass in the value.